cluster/images/kubelet/Makefile

```diff
@@ -1,21 +1,21 @@
-# build the kubelet image.
+# Build the kubelet image.
+#
+# Usage:
+#   VERSION=v1.1.2 [REGISTRY="gcr.io/google_containers"] make build
 
-# TODO: figure out the best base image
-# TODO: figure out how to best source the nsenter binary, if necessary
-# TODO: figure out how to make a single source of version info for these
-#       images
-VERSION=v0.16
```

Comment:
Why not leave a `VERSION=v1.1.3`?
If an argument is specified with make, e.g. `make VERSION=v1.2.0`, `make` will use `v1.2.0`